README.md

```diff
@@ -1,3 +1,43 @@
+# Forked from Zachac/Tweego-File-Splitter
+This is a fork of https://github.com/Zachac/Tweego-File-Splitter with some significant rework done with the help of Copilot.
```

Comment:
I'm not a developer, and since you did the first version, i don't mind if you take all credits, especially since copilot did all the real work, i just gave it some pseudo code. In fact i pull the readme.md in the pull request just because i don't know how to do this with a single file (the py script)😅 
Please feel free to drop my readme or rewrite it if you want.